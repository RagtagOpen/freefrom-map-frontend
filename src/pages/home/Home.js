import React from 'react';
import 'pages/home/Home.css';

function Home() {
    return (
        <div>
            <h1>Home Page</h1>
        </div>
    );
}

export default Home;
