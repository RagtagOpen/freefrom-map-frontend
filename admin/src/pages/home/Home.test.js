import React from 'react';
import renderer from 'react-test-renderer';

import Home from './Home';

it('matches snapshot', () => {
    const home = renderer
        .create(<Home />)
        .toJSON();

    expect(home).toMatchSnapshot();
})
