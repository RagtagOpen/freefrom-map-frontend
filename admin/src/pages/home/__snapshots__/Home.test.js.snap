// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1>
    Home Page
  </h1>
</div>
`;
